extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -25,8 +60,7 @@ private function setWikiaMobileSkin($bool) {
 	 * @dataProvider testRenderInfoboxDataProvider
 	 */
 	public function testRenderInfobox( $input, $expectedOutput, $description ) {
-		$this->setWikiaMobileSkin(false);
-
+		$this->infoboxRenderService = $this->getInfoboxRenderServiceMock( false, $input );
 		$actualOutput = $this->infoboxRenderService->renderInfobox( $input );
 
 		$actualDOM = new DOMDocument('1.0');
```

Comment:
I know you didn't change this line, but I see this block duplicated in the test - any chances to extract the normalizeHTML() function?